src/httpclient.js

```diff
@@ -107,7 +107,8 @@ function create({ baseURL, timeout, headers }) {
       // set body or form based on content type. default is form, except for patch ;-)
       const contentType = myheaders['content-type']
         || (method === 'patch' ? 'application/json' : 'application/x-www-form-urlencoded');
-      if (method && method !== 'get' && method !== 'head') {
+      if (!method || (method !== 'get' && method !== 'head')) {
+        // GET (default) and HEAD requests can't have a body
```

Comment:
yes! you wrong fixed it: `if no method or ... then set body` . that's wrong

Reply:
I noticed and reverted it :)